### PR TITLE
Updated cloud_tops.logic

### PR DESCRIPTION
**Pull request by Nimbus125**

### CODE REVIEW

**Review by CandyCrystal — CHANGES_REQUESTED** [review]
The Zeffa Fluteaways needs to be entrances, and the crest needs do be an exit!

File: cloud_tops.logic [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    #BottomCenterRightPlatform has a tornado that can reach the other platforms w/o items
+    BottomCenterRightPlatform -> BottomCenterPlatform
+    BottomCenterRightPlatform -> BottomCenterLeftPlatform
+    BottomCenterRightPlatform -> BottomLeftPlatform
+    node BottomRightLeftDigItem:
+      chest BottomRightLeftObscureItem
+    BottomCenterRightPlatform -> BottomRightLeftDigItem: item MoleMitts
+    node SmallHole:
+      door SmallHole
+    BottomCenterRightPlatform -> SmallHole: item MoleMitts
```
Comment (Ibot02) [inline cloud_tops.logic:144]: You can also not jump into the hole and go across directly
```suggestion
    BottomCenterRightPlatform -> SmallHole: item MoleMitts
    BottomCenterRightPlatform -> BottomRightCornerPlatform: (item MoleMitts & item Cape)
```

File: cloud_tops.logic [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    #Includes the empty platform to the left of the starting platfirm
+    node BottomRightCornerPlatform:
+      door BottomRightLowerHole
+      door BottomRightUpperHole
+    BottomRightCornerPlatform -> SmallHole: item Cape
+    #Via the center platform
+    BottomRightCornerPlatform -> TopTornadoField: item MoleMitts
+    StartingPlatform -> BottomRightCornerPlatform: item MoleMitts
```
Comment (Ibot02) [inline cloud_tops.logic:152]: This tornado isn't close enough, so you also need cape
```suggestion
    StartingPlatform -> BottomRightCornerPlatform: (item MoleMitts & item Cape)
```

**Review by Ibot02 — CHANGES_REQUESTED** [review]

**Review by CandyCrystal — APPROVED** [review]
Looks good to me!

File: cloud_tops.logic [hunk 3]
```diff
@@ -6,80 +6,71 @@ area CloudTops:
       door CloudTopsExit
       door CenterTornado
       exit BigTornado
-    node CenterFusion:
-      flag CenterFusion
+      lock CenterFusion: key CloudTopsKinstone
     #Reusing the current name for CT Kinstones
```
Comment (Ibot02) [inline cloud_tops.logic:10]: No, you're not :P
```suggestion
```

**Review by Ibot02 — APPROVED** [review]

**Pull request merged**